### ISSUE TITLE
DOES NOT WORK

### ISSUE THREAD

**Issue by SCCOBN** [spookykidmm/zen_installer#24]
I don't know what is is what and how to use it. It does not work. After installation finished OS is not installed.

FIX THIS THING AND MAKE IT WORK.

**Comment by citro333** [comment]
> I don't know what is is what and how to use it. It does not work. After installation finished OS is not installed.
> 
> FIX THIS THING AND MAKE IT WORK.

As it was already mentioned before, the last update was 2 years ago, so it looks like this project is dead. You can use "archfi" instead: https://github.com/MatMoul/archfi

**Comment by ghost** [comment]
@citro333 archfi doesn't work the same.
It doesn't have a nice gui like Zen,it just has a tui.
And you have to start the installer from a normal archiso with a command.
The problem with that is that you have to know how the command to install it was and you would need to connect to wifi via commands that you also have to know.
So you need to know all the commands or another device with a wiki to look at.
For Zen you don't need that.

**Comment by IAmHugh001** [comment]
Exactly. The bullshit for those that think the terminal is the only way to do Arch or any version of Linux needs to end.

**Comment by spookykidmm** [comment]
Hello @Locutus64 @SlushThorsten @SCCOBN @citro333 I'm back, at least momentarily. I just ran an install on my machine and it booted up just fine. I would like to figure out what the issue is. Firstly, I do need to update the iso, but in the meantime if you lot are willing to help me test, here's what you can d
Download archbang iso by my friend Mr. Green here: https://sourceforge.net/projects/archbang/files/ArchBang/archbang-summer-beta-0406-x86_64.iso/download
Boot that
Connect to your wifi network
Follow the instructions under "Fetching and ranking a live mirror list" here: https://wiki.archlinux.org/title/mirrors
Once you have the list of servers, copy that
sudo rm /etc/pacman.d/mirrorlist
sudo nano /etc/pacman.d/mirrorlist
Right click paste
Ctrl-O, Ctrl-X
sudo pacman -S git
git clone https://github.com/spookykidmm/zen_installer
chmod +x zen_installer/zif
sudo zen_installer/zif
Do your install as normal and let me know how it goes

**Comment by glats** [comment]
@spookykidmm 
here is the result http://okturing.com/src/14296/body
